sdk/eventhub/Azure.ResourceManager.EventHubs/tests/Tests/EventHubNamespaceTests.cs

```diff
@@ -376,6 +376,7 @@ public async Task PremiumNamespaceCreateOrUpdate()
 
         [Test]
         [RecordedTest]
+        [Ignore("Depende on KeyVault which will block the pipeline to release new SDK, disable this case temporary")]
```

Comment:
Please have another pr to decouple with the KV dependency first.

Reply:
I will raise another PR to re-enable these cases after this PR merged

Reply:
Here is the remove PR https://github.com/Azure/azure-sdk-for-net/pull/35989